addons/website/views/website_templates.xml

```diff
@@ -267,7 +267,7 @@
 <!-- Header templates -->
 
 <!-- "Default" template -->
-<template id="template_header_default" inherit_id="website.layout" name="Template Header Default" active="True">
+<template id="template_header_default" inherit_id="portal.frontend_layout" name="Template Header Default" active="True" primary="True">
```

Comment:
I think the inherit_id is correct as website.layout is inheriting portal.frontend_layout 
https://github.com/odoo/odoo/blob/410bbee93fff5638950e18d0f1b91c635b894f47/addons/website/views/website_templates.xml#L37
the issue is probably that they need to add primary=True there

introduced change looks breaking for code

Reply:
checked quickly, with primary on Main Layout only error is still there.